apps/rehabstod/vite.config.ts

```diff
@@ -16,7 +16,7 @@ export default ({ mode }: UserConfig) => {
     (result, route) => ({
       ...result,
       [`/${route}`]: {
-        target: process.env.VITE_API_TARGET ?? 'https://rehabstod-devtest.intyg.nordicmedtest.se',
+        target: 'https://rehabstod-devtest.intyg.nordicmedtest.se',
```

Comment:
Should still be possible to change this with `VITE_API_TARGET` environment variable. Use `.env.development.local` to override this instead of changing the configuration.